apps/web/src/services/my-page/components/UserAccountMultiFactorAuthModal.vue

```diff
@@ -131,93 +123,89 @@ watch(() => state.userId, (userId) => {
     <p-button-modal
         :visible="modalState.proxyVisible"
         :header-title="modalState.title"
-        :class="['mfa-modal-wrapper', modalState.proxyType]"
+        class="mfa-modal-wrapper"
         size="sm"
-        :theme-color="modalState.proxyType === 'disabled'? 'alert' : 'primary'"
-        :disabled="modalState.proxyType !== 'change' ? validationState.verificationCode === '' : !(state.isNextStep && state.isSentCode)"
-        :loading="modalState.proxyType !== 'change' ? state.loading : false"
+        :theme-color="props.type === MULTI_FACTOR_AUTH_MODAL_TYPE.DISABLED? 'alert' : 'primary'"
+        :disabled="validationState.verificationCode === '' || !state.isSentCode"
+        :loading="state.loading"
         @confirm="handleConfirmButton"
```

Comment:
suggestion: How about to use `handleClickVerifyButton` here? Or change name of `handleClickVerifyButton` to `handleConfirmButton`?